Field developed processes

waljo11 opened the pull request:
Collection of Field Developed Programs from Broadcom Services

MikeBauerCA commented:
The top-level README still has the following use cases described:
 - Endevor - ISPF tools for Quick-Edit and Endevor
 - Endevor - Package Reporting
 
 Both of these samples were moved under Field Developed Programs. Therefore, the existing links to them are broken.
 
 I would recommend removing these two use cases from the top-level README and replacing them with the Field Developed Programs section you added to the Endevor README.

waljo11 commented:
I made the change you suggested:

   1. Removed the two entries under the top-level README
   2. Updated the entries and fixed the links within the README.md under
   the  Field-Developed-Programs  folder.


On Thu, Apr 13, 2023 at 8:51 AM Mike Bauer ***@***.***> wrote:

> The top-level README still has the following use cases described:
>
>    - Endevor - ISPF tools for Quick-Edit and Endevor
>    - Endevor - Package Reporting
>
> Both of these samples were moved under Field Developed Programs.
> Therefore, the existing links to them are broken.
>
> I would recommend removing these two use cases from the top-level README
> and replacing them with the Field Developed Programs section you added to
> the Endevor README.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/BroadcomMFD/broadcom-product-scripts/pull/36#issuecomment-1507001653>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOA2HB7JDY2MY5A62BF4QA3XBAAG7ANCNFSM6AAAAAAV5RR46Q>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

-- 
This electronic communication and the information and any files transmitted 
with it, or attached to it, are confidential and are intended solely for 
the use of the individual or entity to whom it is addressed and may contain 
information that is confidential, legally privileged, protected by privacy 
laws, or otherwise restricted from disclosure to anyone else. If you are 
not the intended recipient or the person responsible for delivering the 
e-mail to the intended recipient, you are hereby notified that any use, 
copying, distributing, dissemination, forwarding, printing, or copying of 
this e-mail is strictly prohibited. If you received this e-mail in error, 
please return the e-mail to the sender, delete it from your computer, and 
destroy any printed copy of it.
